packages/expect/src/spyMatchers.ts

```diff
@@ -58,14 +52,24 @@ const createToBeCalledMatcher = (matcherName: string) =>
     const pass = count > 0;
     const message = pass
       ? () =>
-          matcherHint('.not' + matcherName, receivedName, '') +
+          matcherHint(matcherName, receivedName, expectedArgument, options) +
           '\n\n' +
-          `Expected ${identifier} not to be called ` +
-          formatReceivedCalls(calls, CALL_PRINT_LIMIT, {sameSentence: true})
+          `Expected number of calls: ${printExpected(0)}\n` +
+          `Received number of calls: ${printReceived(count)}\n\n` +
+          calls
+            .reduce((lines: Array<string>, args: any, i: number) => {
+              if (lines.length < PRINT_LIMIT) {
```

Comment:
Does this still look good for arguments that serialize to large strings?

Reply:
Please see what you think about picture in https://github.com/facebook/jest/pull/8649#issuecomment-509270828